testdata/cs/expected/pkg/outputtest/services.cs

```diff
@@ -0,0 +1,20 @@
+// Code generated by @apexlang/codegen. DO NOT EDIT.
+
+using System;
+
+namespace Urlshortener {
+  public class ShortenerImpl : Shortener {
+    RepositoryImpl repository = new RepositoryImpl();
```

Comment:
This should be in a constructor that takes in the `Repository` interface.

Reply:
Ok